src/components/Dropdown/style.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+
+export const TestFilterWrapper = styled.div`
+  position: relative;
+`;
+
+export const ArrowWrapper = styled.div`
+  display: flex;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  align-items: center;
+  padding-right: 12px;
+  font-size: var(--font-size-12);
+  color: var(--color-grey-80);
+`;
+
+export const TestFilterButton = styled.button`
```

Comment:
Same as above. If possible please create separate style files for individual components.